internal/db/schema/postgres/postgres.go

```diff
@@ -52,6 +52,8 @@ const (
 var defaultMigrationsTable = "boundary_schema_version"
 
 // Postgres is a driver usable by a boundary schema manager.
+// Postgres is not thread safe to be thread safe it should separate the schema access lock id used for internal
+// operations and the one exposed externally.
```

Comment:
Is this a comment on the code or on postgres itself? If the former is there a way to (eventually) fix it?

Reply:
This is a comment on this code.  I had intended for the comment after to indicate 1 way that this code could be fixed if we wanted it to.  Since we use it only inside the manager, which should capture a lock before running it might not make sense to make this thread safe.  I'll simplify the comment to just be "This struct is not thread safe."